account_move_obc_csv/report/report_account_move_obc_csv.py

```diff
@@ -177,12 +179,24 @@ def _get_report_vals_dict(self, record):
 
     def generate_csv_report(self, writer, data, records):
         self._check_records(records)
+        # Sort records by date and then by picking_id
+        sorted_records = sorted(
+            records, key=lambda r: (r.date, r.stock_move_id.picking_id.id)
+        )
         writer.writeheader()
-        for record in records:
-            vals_dict = self._get_report_vals_dict(record)
-            for _k, v in sorted(vals_dict.items()):
-                writer.writerow(v)
-            record.is_exported = True
+        # Group records by picking_id
+        for _p, grouped_records in groupby(
+            sorted_records, key=lambda r: r.stock_move_id.picking_id.id
+        ):
+            first_record = True
+            for record in grouped_records:
+                vals_dict = self._get_report_vals_dict(record)
+                if first_record:
+                    vals_dict[1]["GL0010000"] = "*"
+                    first_record = False
+                for _k, v in sorted(vals_dict.items()):
+                    writer.writerow(v)
+                record.is_exported = True
```

Comment:
We need to consider the non-inventory journal entries (e.g. those from vendor bills).  The first line need to be marked with `*` for those as well.
```suggestion
        picking = self.env["stock.picking"]
        for record in sorted_records:
            vals_dict = self._get_report_vals_dict(record)
            first_record = False
            record_picking = record.stock_move_id.picking_id
            if not record_picking:
                first_record = True
            elif picking != record_picking:
                first_record = True
                picking = record_picking
            if first_record:
                vals_dict[1]["GL0010000"] = "*"
            for _k, v in sorted(vals_dict.items()):
                writer.writerow(v)
            record.is_exported = True
```